airbyte/_executors/declarative.py

```diff
@@ -99,9 +99,10 @@ def execute(
         args: list[str],
         *,
         stdin: IO[str] | AirbyteMessageIterator | None = None,
+        env: dict[str, str] | None = None,
     ) -> Iterator[str]:
         """Execute the declarative source."""
-        _ = stdin  # Not used
+        _ = stdin, env  # Not used
```

Comment:
TODO: We may need to (later, not here) implement a cache option for declarative execution. This runs through our same Python process, so we may either need to patch the `requests` library or have another implementation. Lmk if you have thoughts, but don't implement.